app/controllers/jobs_controller.rb

```diff
@@ -1,12 +1,35 @@
 class JobsController < ApplicationController
+
+  def dashboard
+    if params[:sort]
+      @sort = params[:sort]
```

Comment:
no need to set this as an instance variable if we are just using it on the next line 👍